### ISSUE TITLE
Set hidden attributes is not working properly

### ISSUE THREAD

**Issue by bankorh** [arangodb/arangodb-php#182]
OS : `Mac OS X Yosemite 10.10.4`
PHP : `5.6.2`
ArangoDB : `2.6.1`
arangodb-php ：`2.5`

Recently I have done something to retrieve a document from the database.

This is the record in the database.
```json
{
	"isActive":true,
	"username":"bankorh",
	"password":"123"
}
```
And these are my codes.
```php
// getDocument() is my function to get the data.
$document = getDocument($username);

// set the hidden attributes for this document.
$document->setHiddenAttributes(array(
	'password'
));

// get all the data in the document.
$result = $document->getAll();
```
The `password` should not show up in result because I have set it as hidden attribute.
But the `password` appeared because of `isActive` is set to true.
However, if I change the `isActive` to false, the `password` will be hidden again.

**Comment by frankmayer** [comment]
Thanks for reporting. Will look into it.

**Comment by jsteemann** [comment]
@frankmayer : currently looking into it as well and having it fixed locally. will commit soon.

**Comment by frankmayer** [comment]
ok :+1:

**Comment by bankorh** [comment]
Thank you. =D

**Comment by jsteemann** [comment]
Fixed in 2.5, 2.6 and devel branches.
